content/en/platform/corda/5.2/deploying-operating/deployment/upgrading521/_index.md

```diff
@@ -0,0 +1,517 @@
+
+```
+kafka-topics --bootstrap-server=prereqs-kafka.test-namespace:9092 --list
+```
+
+### REST API Access
+
+To get information about TLS certificate aliases, the data migration step requires access to the Corda REST API. This is required at the same stage when data is extracted from Kafka, which means access to Kafka and the Corda REST API must be available in the same environment at the same time. You must pass credentials and locations of both to the Corda CLI plugin when executed.
+
+Also, you must have `GET` access to the `/key` and `/certificate` APIs.
+
+### Corda CLI Tool and Corda 5 Plugins
+
+Some upgrade steps use the {{< tooltip >}}Corda CLI{{< /tooltip >}} tool used also in Corda 5 [manual bootstrapping]({{< relref "../deploying/manual-bootstrapping.md" >}}). Users who entirely bootstrapped Corda 5 using the Corda Helm chart may not be familiar with this tool because the Corda Helm chart executes it for you. For instructions on how to install Corda CLI, go to [Installing the Corda CLI]({{< relref "../../../application-networks/tooling/installing-corda-cli.md" >}}).
+
+The Corda CLI tool and plugins are also published internally as Docker images. The examples in this script use those Docker images so that:
+* The Corda CLI tool does not need to be downloaded or installed.
+* The latest published tool is always available.
+* The correct version is always used.
+
+{{< note >}}
+The version of the Corda CLI tool and plugins must match the version of Corda 5 being upgraded to, that is, 5.2.1.
+{{< /note >}}
+
+The functionality of the tools, the database schemas and Kafka topics embedded in them pertain to the version of Corda they were built against.
+
+If you are not using Docker images, where you come across an example of the form:
+```
+docker run <docker options> $CLI_DOCKER_IMAGE <tool options>
+```
+
+This can be replaced with:
+```
+corda-cli.sh <tool options>
+```
+
+To use the Docker images, you must set `CLI_DOCKER_IMAGE` to the 5.2.1 image version, for example, `corda-os-docker.software.r3.com/corda-os-plugins:5.2.1.0` or some beta version prior to 5.2.1 release.
+
+## Back Up the Corda Database
+
+You must create a backup of all schemas in your database:
+
+* Cluster — name determined at bootstrap. For example, `CONFIG`.
+* Crypto — name determined at bootstrap. For example, `CRYPTO`.
+* RBAC — name determined at bootstrap. For example, `RBAC`.
+* Virtual nodes schemas — multiple schemas per virtual node holding ID:
+   * `vnode_crypto_<holding_id>`
+   * `vnode_uniq_<holding_id>`
+   * `vnode_vault_<holding_id>`
+
+How these backups are performed, stored, and managed depends on their owner. It differs based on the database service supplier, and it is out of scope of this guide to detail that. Please refer to the database service provider’s documentation.
+
+5.2.1 migration only covers the cluster schema, backing up everything else is a precaution.
```

Comment:
Might be better written as `5.2.1 migration only makes changes in the cluster schema...`